### PR TITLE
[CN-1372]: add note to TLS document about how to update TLS

### PR DESCRIPTION
**Pull request by kutluhanmetin**

### CODE REVIEW

**Review by hasancelik — APPROVED** [review]

File: docs/modules/ROOT/pages/tls.adoc [hunk 1]
```diff
@@ -74,3 +74,5 @@ include::ROOT:example$/hazelcast-tls.yaml[]
 ----
 include::ROOT:example$/management-center-tls.yaml[]
 ----
+
+NOTE: To update TLS certificates, it is required to delete and recreate the Hazelcast/Management Center custom resource.
```
Comment (oliverhowell) [inline docs/modules/ROOT/pages/tls.adoc:78]:
```suggestion
NOTE: To update TLS certificates, you need to delete and recreate the Hazelcast/Management Center custom resource.
```
Is it either the Hazelcast or MC resource you need to delete or both? The / is ambiguous here...

**Review by oliverhowell — CHANGES_REQUESTED** [review]

File: docs/modules/ROOT/pages/tls.adoc (hunk 1 above)
Comment (kutluhanmetin) [reply docs/modules/ROOT/pages/tls.adoc:78]: TLS certificates are applicable for both CRs. So whichever you created you should delete it and recreate.

Comment (oliverhowell) [reply docs/modules/ROOT/pages/tls.adoc:78]: What do you think of update suggestion then?

**Review by oliverhowell — APPROVED** [review]
Approving in advance

File: docs/modules/ROOT/pages/tls.adoc (hunk 1 above)
Comment (kutluhanmetin) [reply docs/modules/ROOT/pages/tls.adoc:78]: I think this should be enough: `NOTE: To update TLS certificates, you need to delete and recreate the appropriate custom resource`, WDYT?

Comment (oliverhowell) [reply docs/modules/ROOT/pages/tls.adoc:78]: Yep, ship it!

**Pull request merged**